__DEFINES/setup.dm

```diff
@@ -1700,7 +1702,7 @@ var/proccalls = 1
 #define ESPORTS_CULTISTS "Team Geometer"
 #define ESPORTS_SECURITY "Team Security"
 
-#define DNA_SE_LENGTH 58
```

Comment:
Shouldn't this be 59?

Reply:
@robintime forgot to make it that in her anemia PR so I added 2 to cover it instead. (it was causing a warning on server bootups)